core/src/main/java/com/hcl/ecomm/core/services/impl/package-info.java

```diff
@@ -0,0 +1,19 @@
+/*
+ *  Copyright 2015 Adobe Systems Incorporated
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+@Version("1.0")
```

Comment:
@rakeshkumar-verma : do we need package-info.java inside impl too is we already have it in core/src/main/java/com/hcl/ecomm/core/services/package-info.java ??

Reply:
@rishabh-behl  not required. deleted.